llvm/test/DebugInfo/Generic/assignment-tracking/dse/dse-after-memcpyopt-merge.ll

```diff
@@ -1,3 +1,4 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
```

Comment:
Not a full review (sorry), just looking at this debug-info test update specifically. Please can you avoid using `update_test_checks.py` on this test?

Generally we've aimed for debug-info tests to be fairly targeted, without additional noise, so that it's easier to see what's going on and to reduce churn in the test as a result of unrelated non-debug changes to the IR. There's been discussion previously as to whether some could use `update_test_checks.py` but IIRC the current consensus is to leave the debug info tests as manual ones - if this approach is causing problems IMO it should be tackled at a higher level (RFC / discussion on Discourse).

Separately, IMO if we were to change a test to use `update_test_checks.py` I'd feel more comfortable if that was done in a separate NFC patch.

Reply:
Does seem to be failing anymore with the change so dropped all changes to `DebugInfo/Generic/assignment-tracking/dse/dse-after-memcpyopt-merge.ll`